### PR TITLE
Update XDC Network info.

### PR DESCRIPTION
**Pull request by AnilChinchawale**
- Add extra rpcs in Mainnet
- update explorer links
- add the faucet link in XDC Apothem/TestNet

### CODE REVIEW

**Comment by gustavoelb17** [comment]
> - Add extra rpcs in Mainnet
> 
> - update explorer links
> 
> - add the faucet link in XDC Apothem/TestNet

**Comment by trastvolt** [comment]
AnilChinchawale:master

File: _data/icons/blocksscan.json [hunk 1]
```diff
@@ -0,0 +1,8 @@
+[
+    {
+      "url": "ipfs://QmPzVFs16GwaD8LAcGFLCNXzEK8BHFKNXeM3nmBpnq9xy3",
```
Comment (ligi) [inline _data/icons/blocksscan.json:3]: I can resolve the other CID - but not this one - where did you pin it?

Comment (AnilChinchawale) [reply _data/icons/blocksscan.json:3]: Cloudflare IPFS :-  https://cloudflare-ipfs.com/ipfs/QmPzVFs16GwaD8LAcGFLCNXzEK8BHFKNXeM3nmBpnq9xy3

**Review by ligi — APPROVED** [review]

**Pull request merged**